inspirehep/dojson/hep/fields/bd6xx.py

```diff
@@ -78,9 +78,20 @@ def free_keywords2marc(self, key, value):
 @utils.filter_values
 def accelerator_experiments(self, key, value):
     """The accelerator/experiment related to this record."""
+    recid = ''
+    curated_relation = False
+    if '0' in value:
+        try:
+            recid = int(value.get('0'))
+        except:
```

Comment:
Instead of blind `except: pass`, I have had previous success with specifying the most common exceptions:

```python
except (TypeError, ValueError, AttributeError):
    pass
```